vmdb/app/helpers/container_service_helper/textual_summary.rb

```diff
@@ -53,12 +52,16 @@ def textual_portal_ip
     {:label => "Portal IP", :value => @record.portal_ip}
   end
 
-  def textual_protocol
-    {:label => "Protocol", :value => @record.protocol}
-  end
 
-  def textual_container_port
-    {:label => "Container Port", :value => @record.container_port}
+  def textual_port_config(single_conf)
+    name = single_conf.name
+
+    name = "Unnamed" if name.nil? || name.empty?
```

Comment:
This can be simplified to `name = "Unnamed" if name.blank?`

@h-kataria @dclarizio Do you have some common string you use for unnamed things to be consistent elsewhere in the UI?